src/api/app/views/layouts/webui2/_footer.html.haml

```diff
@@ -29,6 +29,9 @@
         %a{ href: 'http://en.opensuse.org/Portal:Build_service' } OBS Portal
       %li
         %a{ href: 'http://openbuildservice.org/support/' } Reporting a Bug
+      - if Announcement.last.present?
+        %li
+          %a{ href: announcement_path(Announcement.last) } Terms of Service
```

Comment:
why not `link_to` instead of `%a`? :thinking:

Reply:
Good question :thinking: We've been just following the existing style in this file.

Reply:
> Any advice?

Always use link_to! :-)

> I was just wondering if I had to change everything or imitate and wait for refactoring.

Always do `the right thing[tm]`. Waiting for unknowns is only `the right thing[tm]` if it's a dangerous situation :clown_face:

Reply:
Same here: I would use a more generic name for the link!

Reply:
> hennevogel 18 hours ago  Member
> > Any advice?

> Always use link_to! :-)

I agree, but let's do that in a seperate PR.

Reply:
@bgeuken, fine for me, let's do it in a separate PR :smile_cat: